### ISSUE TITLE
Getting the message "Slim Application Error" in browser after installation (Ubuntu 20.4)

### ISSUE THREAD

**Issue by LOIZoclock** [AzuraCast/AzuraCast#5227]
### Installation Method

Ansible Installation

### AzuraCast Release Channel

Rolling Release Channel

### What happened?

I have installed Azuracast "Ansible" by using the official commands (with git clone).

When I am visiting the ip on my browser I get the message "Slim Application Error".

Please note that installation finished without failed tasks but with 3 ignored tasks, noted below.

### Relevant log output

```shell
[...]

TASK [supervisord : Kill legacy radio processes (if upgrading from older version)] *******************************************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["killall", "-q", "-9", "icecast2", "sc_serv", "liquidsoap"], "delta": "0:00:00.00ero return code", "rc": 1, "start": "2022-03-23 20:50:58.437065", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
...ignoring

[...]

TASK [php : Remove Old PHP Libraries] ****************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "No package(s) matching 'php5-*' available"}
...ignoring

[...]

TASK [azuracast-db-install : MariaDB Cleanup] ********************************************************************************************
ok: [localhost] => (item=DELETE FROM mysql.user WHERE User='')
ok: [localhost] => (item=DELETE FROM mysql.user WHERE User='root' AND Host NOT IN ('localhost', '127.0.0.1', '::1'))
failed: [localhost] (item=DROP DATABASE test) => {"ansible_loop_var": "azure_db_install_sql_cleanup_query", "azure_db_install_sql_cleanup_mysql", "--defaults-extra-file=/root/.my.cnf", "-ne", "DROP DATABASE test"], "delta": "0:00:00.008548", "end": "2022-03-23 20:52:34.242327-03-23 20:52:34.233779", "stderr": "ERROR 1008 (HY000) at line 1: Can't drop database 'test'; database doesn't exist", "stderr_lines": ["Edatabase doesn't exist"], "stdout": "", "stdout_lines": []}
ok: [localhost] => (item=DELETE FROM mysql.db WHERE Db='test' OR Db='test\_%')
...ignoring

[...]

localhost                  : ok=100  changed=78   unreachable=0    failed=0    skipped=10   rescued=0    ignored=3
```

**Comment by SlvrEagle23** [comment]
@LOIZoclock I believe you can view logs in `/var/azuracast/www_tmp`...check that directory, it should have an application log in it with details on the error you're experiencing.

**Comment by webafrancois** [comment]
@LOIZoclock I make a fresh install from rolling release too this night, with Ansible on a fresh ubuntu 20.04 server.
I've the same problem.

Try to run an ./update.sh from your /var/azuracast/www folder. 
In my case, it works ! And I can reach the admin page and configure My Azuracast in Rolling Release #[21675cd] (2022-03-27 1:04) • Ansible • PHP 8.1

But Azuracast works without sound (on a stable install too) in default public page : 
http://ensemble.cemea.org/public/webradios_ensemble

my /var/azuracast/www_tmp/app-2022-03-27.log is attached.

But the direct url works : http://ensemble.cemea.org:8000/radio.mp3

[app-2022-03-27.log](https://github.com/AzuraCast/AzuraCast/files/8358308/app-2022-03-27.log)

**Comment by webafrancois** [comment]
Same log from fresh install from stable, with only the doc script : git clone, checkout stable and install.
show a : 
Slim Application Error
A website error has occurred. Sorry for the temporary inconvenience.

[stable-0.15.2-app-2022-03-27.log](https://github.com/AzuraCast/AzuraCast/files/8358467/stable-0.15.2-app-2022-03-27.log)

And now , with a update.sh
[stable-update-0.15.2-app-2022-03-27.log](https://github.com/AzuraCast/AzuraCast/files/8358470/stable-update-0.15.2-app-2022-03-27.log)

But now, it works : 
* first login force to create a webradio
with sound :+1: :heart: 

I'm using the commit : 
commit d77f624eebf926041a9a2461c6090e99244081b4 (HEAD -> stable, origin/stable)
Author: ogrenci01 <ogrenci01@gmail.com>
Date:   Sat Feb 26 23:28:38 2022 +0300

    Update liquidsoap.yml (#5160)
    
    (cherry picked from commit ebb5c7d13248c1d0b6f4d52d67d62938561ccb81)


PS : The url must begin with http, or domain alone return an error like : 
```
MalformedUriException at /var/azuracast/www/vendor/guzzlehttp/psr7/src/Uri.php L730: The path of a URI with an authority must start with a slash "/" or be empty
``` 

I've an error on picture song with a nginx error : 
```
403 Forbidden
nginx
```

**Comment by webafrancois** [comment]
And in /var/azuracast/www_tmp/error.log some error from nginx : 
```
2022/03/27 21:10:55 [error] 119208#119208: *12 open() "/var/azuracast/stations/webradio_cemea/media/.albumart/019482c60b1864841d4ae383.jpg" failed (13: Permission denied), client: 92.184.116.102, server: localhost, request: "GET /api/station/1/art/019482c60b1864841d4ae383-1648414311.jpg HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "webradios.cemea.org", referrer: "http://webradios.cemea.org/station/1/profile"
2022/03/27 21:10:56 [error] 119208#119208: *12 open() "/var/azuracast/stations/webradio_cemea/media/.albumart/22f68de543edebb5abd0de93.jpg" failed (13: Permission denied), client: 92.184.116.102, server: localhost, request: "GET /api/station/1/art/22f68de543edebb5abd0de93-1648414312.jpg HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "webradios.cemea.org", referrer: "http://webradios.cemea.org/station/1/profile"
2022/03/27 21:10:58 [error] 119208#119208: *12 open() "/var/azuracast/stations/webradio_cemea/media/.albumart/019482c60b1864841d4ae383.jpg" failed (13: Permission denied), client: 92.184.116.102, server: localhost, request: "GET /api/station/1/art/019482c60b1864841d4ae383-1648414311.jpg HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "webradios.cemea.org", referrer: "http://webradios.cemea.org/public/webradio_cemea"
2022/03/27 21:12:45 [error] 119208#119208: *12 open() "/var/azuracast/stations/webradio_cemea/media/.albumart/22f68de543edebb5abd0de93.jpg" failed (13: Permission denied), client: 92.184.116.102, server: localhost, request: "GET /api/station/1/art/22f68de543edebb5abd0de93-1648414312.jpg HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "webradios.cemea.org", referrer: "http://webradios.cemea.org/station/1/profile"
2022/03/27 21:12:45 [error] 119208#119208: *12 open() "/var/azuracast/stations/webradio_cemea/media/.albumart/5b88c827ff17882ff4a92ef2.jpg" failed (13: Permission denied), client: 92.184.116.102, server: localhost, request: "GET /api/station/1/art/5b88c827ff17882ff4a92ef2-1648414309.jpg HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "webradios.cemea.org", referrer: "http://webradios.cemea.org/station/1/profile"
```

**Comment by webafrancois** [comment]
Nginx error in albumart is solved in #5248 whith good permissions.
Maybe a patch in stable version ?
